CONTRIBUTING.md

```diff
@@ -5,8 +5,21 @@ email, or any other method with the owners of this repository before making a ch
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
+## First things
+If you are new to contributing please follow this [guide](https://github.com/firstcontributions/first-contributions#first-contributions).
+
+We recommend you choose a [good first issue](https://github.com/rubatopy/rubato/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22); these issues have been identified as requiring a less in depth understanding of rubato to close.
+We tend to review PRs within a day or two, but feel free to ping us on our [discord](https://discord.gg/rueSETyRKx).
+
+After having contributed to the project you will be put on the CONTRIBUTORS.md document. Thank you for your help!
+
 ## Setting up dev environment
 
+Notes:
+* we are a 3.10 python library, you will need this installed.
+* We recommend using visual studios and installing all of our extensions, this will commit your code with the correct formatting and type checking.
```

Comment:
```suggestion
* We recommend using [Visual Studio Code](https://code.visualstudio.com/) and installing all of our extensions, this will commit your code with the correct formatting and type checking.
```